test/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDaoTest.java

```diff
@@ -53,7 +56,7 @@
 
     private final StudyIdentifier studyIdentifier = new StudyIdentifierImpl(TEST_STUDY_IDENTIFIER);
     private TestSurvey testSurvey;
-    private Set<GuidCreatedOnVersionHolderImpl> surveysToDelete;
+    private Set<GuidCreatedOnVersionHolder> surveysToDelete;
```

Comment:
Semantically, we want this set to only have immutable HolderImpls and not Surveys, so HolderImpl is correct here.